src/main/java/com/metsci/laproc/display/BasicWindow.java

```diff
@@ -0,0 +1,86 @@
+package com.metsci.laproc.display;
+
+import com.jogamp.opengl.util.FPSAnimator;
+import com.metsci.glimpse.canvas.NewtSwingGlimpseCanvas;
+import com.metsci.glimpse.docking.*;
+import com.metsci.laproc.ROCCurvePlot;
+import com.metsci.laproc.plotting.Axis;
+import com.metsci.laproc.plotting.BasicGraph;
+import com.metsci.laproc.plotting.Curve;
+import com.metsci.laproc.plotting.Graph;
+
+import javax.swing.*;
+import java.awt.*;
+
+/**
+ * Created by porterjc on 9/22/2016.
+ */
+public class BasicWindow implements Window{
+    private GraphPanel graphPanel = new GraphPanel();
+
+    public void display() {
+       // MultiSplitPane pane = new MultiSplitPane(50);
+
+        DockingGroup group = new DockingGroup(DockingThemes.defaultDockingTheme, DockingGroup.DockingFrameCloseOperation.DISPOSE_CLOSED_FRAME);
```

Comment:
On close, I get this error:
Sep 29, 2016 10:20:10 AM com.metsci.glimpse.docking.DockingGroup$1 windowClosing
WARNING: Refusing to dispose frame, because it contains uncloseable views